Src/Support/IntegrationTests/ServiceAccountTests.cs

```diff
@@ -74,19 +74,51 @@ public async Task JwtAccessToken_Http()
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
-        [Fact]
-        public void JwtAccessToken_RestClient()
+        [Theory]
+        [InlineData(true, null)]
+        [InlineData(false, null)]
+        [InlineData(true, new[] { "https://www.googleapis.com/auth/bigtable.admin" })]
+        public void JwtAccessToken_RestClient(bool useJwtAccessWithScopes, string[] scopes)
         {
             // See: https://developers.google.com/identity/protocols/oauth2/service-account#jwt-auth
 
+            GoogleCredential credential = useJwtAccessWithScopes 
+                ? Helper.GetServiceCredentialWithJwtFlag().CreateScoped(scopes) 
+                : Helper.GetServiceCredential().CreateScoped(scopes);
+
             BigtableAdminService service = new BigtableAdminService(new BaseClientService.Initializer
             {
-                HttpClientInitializer = Helper.GetServiceCredential().CreateScoped()
+                HttpClientInitializer = credential
             });
             
             ListInstancesResponse response = service.Projects.Instances.List($"projects/{Helper.GetProjectId()}").Execute();
 
             Assert.NotNull(response);
         }
+
+        [Fact]
+        public void JwtAccessToken_WrongScope_RestClient()
+        {
+            // See: https://developers.google.com/identity/protocols/oauth2/service-account#jwt-auth
+
+            GoogleCredential credential = Helper.GetServiceCredentialWithJwtFlag().CreateScoped(new[] { "https://www.googleapis.com/auth/admob.readonly" });
+
+            BigtableAdminService service = new BigtableAdminService(new BaseClientService.Initializer
+            {
+                HttpClientInitializer = credential
+            });
+
+            string expectedError = "Request had invalid authentication credentials.";
+
+            try
+            {
+                ListInstancesResponse response = service.Projects.Instances.List($"projects/{Helper.GetProjectId()}").Execute();
+                throw new Exception("should fail");
+            }
+            catch (Exception ex)
+            {
+                Assert.Contains(expectedError, ex.Message);
+            }
```

Comment:
```suggestion
            var exception = Assert.Throws<GoogleApiException>(() =>
                service.Projects.Instances.List($"projects/{Helper.GetProjectId()}").Execute());
            Assert.Contains(expectedError, exception.Message);
```

Reply:
thanks :) i was looking for a junit alternative but missed it